ajax/libs/chartkick/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "chartkick",
+  "filename": "chartkick.min.js",
+  "version": "1.4.1",
+  "homepage": "https://github.com/ankane/chartkick.js",
+  "description": "Create beautiful Javascript charts with minimal code",
+  "keywords": [
+    "charts"
+  ],
+  "authors": [
```

Comment:
@Piicksarn 
There is only one author, I think using "author" field with a string would be nicer.

And, could you give this library more keywords? Thanks. :-)